mne/viz/topomap.py

```diff
@@ -614,6 +618,9 @@ def __init__(self, pos, extrapolate, origin, radii, border):
         self.mask_pts = mask_pts
         self.border = border
         self.tri = tri
+        self.interp = {'cubic': CloughTocher2DInterpolator,
+                       'nearest': CloughTocher2DInterpolator,  # hack not used
```

Comment:
if it is not used then I'd remove it. :)

Reply:
I don't understand - the `_GridData` object shouldn't be used when `image_interp='nearest'`.

Reply:
You need the extent which is in the `_setup_interp` method, which is not easy to untangle